src/service/action-impl.js

```diff
@@ -85,6 +85,9 @@ export class ActionService {
   constructor(win) {
     /** @const {!Window} */
     this.win = win;
+
+    // Add core events.
+    this.addEvent('tap');
```

Comment:
Make an installEvents method, so the caller doesn't need to know about what is being installed?